hadoop-hdds/docs/content/feature/_index.zh.md

```diff
@@ -21,13 +21,7 @@ weight: 4
     limitations under the License.
     -->
 
-{{<jumbotron title="GDPR 在 Ozone 中的应用">}}
-          通用数据保护条例（GDPR）是关于如何处理个人数据的规范，它属于欧盟的法律，但是由于软件常常跨越地理位置传播和使用，Ozone 支持 GDPR 中的删除权（被遗忘权）。
+{{<jumbotron title="Ozone 特性">}}
+        以下页面介绍 Ozone 的相关特性，比如高可用性、拓扑感知和指标工具等。
```

Comment:
minor problem: 
the translation of this sentence seems to be missing compared to the [original content](https://github.com/apache/ozone/blob/master/hadoop-hdds/docs/content/feature/_index.md).

Can it be more perfect? But it does not affect the overall semantic meaning.

Reply:
Could you kindly point out the specific missing part ? Please be aware that it may not always be possible to translate word by word. Really appreciate if better expression.

Reply:
Translate into something like "Ozone特性在下面几个页面解释。比如高可用性、拓扑感知、指标工具等特性将在下一页面中详细介绍"

It seems to be more complete.

Reply:
"Ozone 特性在下面几个页面解释。比如高可用性、拓扑感知、指标工具等特性将在下一页面中详细介绍。"

I agree that this is the direct translation. However, `下面几个页面` and `下一页面中` are redundant, `解释` and `详细介绍` are redundant, which are not the normal way that we would speak in Chinese, right ?

Reply:
Yes, it does not read smoothly, you can leave it as it is~ (just a suggestion)